src/GTreePluto.cc

```diff
@@ -42,6 +42,37 @@ GTreePluto::ParticleList GTreePluto::GetAllParticles() const
     return list;
 }
 
+PParticle* GTreePluto::GetMCTrue(const int idx) const
+{
+    if (!PlutoMCTrue) {
+        //return nullptr;
+        fprintf(stderr, "ERROR: No Pluto Tree in current file!\n");
+        exit(1);
+    }
+
+    return dynamic_cast<PParticle*>(PlutoMCTrue->At(idx));
+}
+
+TLorentzVector GTreePluto::GetTrueP4(const int idx) const
+{
+    if (!PlutoMCTrue) {
+        fprintf(stderr, "ERROR: No Pluto Tree in current file!\n");
+        exit(1);
```

Comment:
This is bad.

GTrees should never end the program.
Better return nothing or "zero" of some sort. If not possible or unlogical, use throw to "throw" an exception with a suitable error message.